yktrace/src/tir.rs

```diff
@@ -53,13 +63,13 @@ lazy_static! {
 /// The serialised IR loaded in from disk. One of these structures is generated in the above
 /// `lazy_static` and is shared immutably for all threads.
 pub struct Sir {
-    /// Lets us map a DefId from a trace to a SIR body.
-    pub bodies: HashMap<DefId, Body>,
+    /// Lets us map a symbol name to a SIR body.
+    pub bodies: HashMap<String, Body>,
     // Interesting locations that we need quick access to.
     pub markers: SirMarkers
 }
 
-/// Contains the DefIds of interesting locations required for trace manipulation.
+/// Records interesting locations required for trace manipulation.
```

Comment:
"interesting" is very vague. Can we be mildly more specific?

Reply:
The locations are explained in detail a few lines down. I feel expanding this comment would only duplicate.

Reply:
Can we just remove the word "interesting" then?